ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -4733,3 +4733,12 @@ HWRWISOVPM:
   - heating_return_water_isolation_valve_percentage_sensor
   implements:
   - MONITORING
+
+RDM:
```

Comment:
RDC would be more appropriate, because you are commanding the return damper. Please rename.